tree/ntupleutil/v7/inc/ROOT/RNTupleInspector.hxx

```diff
@@ -56,17 +57,78 @@ std::cout << "The compression factor is " << std::fixed << std::setprecision(2)
 */
 // clang-format on
 class RNTupleInspector {
+public:
+   /// Holds column-level storage information.
+   class RColumnInfo {
+   private:
+      const RColumnDescriptor &fColumnDescriptor;
+      std::uint64_t fOnDiskSize = 0;
+      std::uint64_t fInMemorySize = 0;
```

Comment:
This is just `fElementSize` * `fElements`, isn't it? In this case, I'd make this a computed quantity so that we can never have inconsistent information.